sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1014,6 +1035,25 @@ OPTION: 'OPTION';
 ANTI: 'ANTI';
 LOCAL: 'LOCAL';
 INPATH: 'INPATH';
+AUTHORIZATION: 'AUTHORIZATION';
+CHECK: 'CHECK';
```

Comment:
When ansi is off, we need to ensure all these newly added keywords are still not reserved. Thus, `CHECK` should be added to `nonReserved `, right?

Reply:
I checked the newly added keywords. Below is the list that are not included:

```
 CHECK: 'CHECK';
 COLLATE: 'COLLATE';
 OVERLAPS: 'OVERLAPS';
 ONLY: 'ONLY';
 PRIMARY: 'PRIMARY';
 REFERENCES: 'REFERENCES';
 UNIQUE: 'UNIQUE';
 VARYING: 'VARYING';
```

Reply:
Yes, you're right, I'll update